pt-BR/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -68,7 +68,7 @@
 12509=Montando mensagem…
 
 ## @name NS_MSG_NO_SENDER
-12510=Nenhum remetente foi especificado. Por favor, preencha seu endereço de email em “Configurar contas” no menu “Ferramentas”.
+12510=Nenhum remetente foi especificado. Por favor, preencha seu endereço de e-mail em “Configurar contas” no menu “Ferramentas”.
 
 ## @name NS_MSG_NO_RECIPIENTS
 12511=Nenhum destinatário foi especificado. Por favor, selecione um destinatário ou newsgroup no campo de endereçamento.
```

Comment:
CORRECTIONS NEEDED it should be 12511=Nenhum destinatário foi especificado. Por favor, selecione um destinatário ou grupo de notícias no campo de endereçamento.